sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -261,13 +265,21 @@ def build_list(state, arg):
 
 
 def load_attr(state, arg):
+  """Replaces the top of the stack, TOS, with with
+  getattr(TOS, co_names[arg])
+  """
   o = state.stack.pop()
   name = state.get_name(arg)
   if isinstance(o, Const) and hasattr(o.value, name):
     state.stack.append(Const(getattr(o.value, name)))
-  elif (isinstance(o, type)
-        and isinstance(getattr(o, name, None), types.MethodType)):
-    state.stack.append(Const(BoundMethod(getattr(o, name))))
+  elif (inspect.isclass(o) and
```

Comment:
<!--new_thread; commit:b015963f671ceb39f8033014eb6d4e46a4a734bc; resolved:0-->
Does `inspect.isclass(o)` do the same thing as `isinstance(o, type)` for python 2?

Reply:
inspect.isclass(o) is isinstance(o, (type, types.ClassType)) which accounts for old-style classes as well. This seems to be a desired function according to trivial_inference.py. There are no test cases for because of pylint. This might be considered as a separate issue.